editor/components/block-list/block-draggable.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { Draggable } from '@wordpress/components';
+
+function BlockDraggable( { rootUID, index, uid, layout, isDragging, ...props } ) {
+	const className = classnames( 'editor-block-list__block-draggable', {
+		'is-visible': isDragging,
+	} );
+
+	const transferData = {
+		type: 'block',
+		rootUID: rootUID,
+		fromIndex: index,
+		uid: uid,
+		layout,
```

Comment:
Minor: Inconsistency: We use object property shorthand for some, but not all properties. `uid` and `rootUID` qualify as well.